internal/controllers/scheduling/controller.go

```diff
@@ -125,13 +125,26 @@ func (c *controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 			continue
 		}
 
+		// Skip operations for paused synthesizers
+		if synth.Spec.Paused {
+			continue
+		}
+
 		next := newOp(&synth, &comp, nextSlot)
 		if next != nil && (op == nil || next.Less(op)) {
 			op = next
 		}
 	}
 	freeSynthesisSlots.Set(float64(c.concurrencyLimit - inFlight))
 
+	// Update synthesizer status for all synthesizers
+	for _, synth := range synths.Items {
+		if err := updateSynthesizerStatus(ctx, c.client, &synth, comps.Items); err != nil {
```

Comment:
It's actually okay to return an error here. We need to make sure the status eventually converges on the expected state.